src/components/schedule-page/table/schedule/schedule-parts/base-cell/use-component-visible.tsx

```diff
@@ -4,18 +4,16 @@
 import { useState, useEffect, useRef } from "react";
 
 /* eslint-disable @typescript-eslint/no-explicit-any */
-/* eslint-disable @typescript-eslint/explicit-function-return-type */
 export default function useComponentVisible(initialIsVisible: boolean): any {
```

Comment:
tutaj w sumie to any można zastąpić tworząc interfejs tego hooka { ref: RefObject, isComponentVisible: boolean, setIsComponentVisible: (oldState: boolean) => void }

Reply:
poprawiłam :)